consensus/polybft/consensus_runtime.go

```diff
@@ -164,9 +164,11 @@ func newConsensusRuntime(log hcf.Logger, config *runtimeConfig) (*consensusRunti
 	var bridgeManager BridgeManager
 
 	if runtime.IsBridgeEnabled() {
-		bridgeManager, err = newBridgeManager(runtime, config, runtime.eventProvider, log)
-		if err != nil {
-			return nil, err
+		for chainID, _ := range config.GenesisConfig.Bridge {
+			bridgeManager, err = newBridgeManager(runtime, config, runtime.eventProvider, log, chainID)
```

Comment:
I would change this. Yes, we will have multiple bridge managers, but let's make a struct like this:

```go
type Bridge []BridgeManager
```

And make it encapsulate the functions that the current `BridgeManager` interface has. It will be an empty slice if blade is L1 (no bridge configured). And if it is, then just have a constructor for it, which will iterate through `config.GenesisConfig.Bridge` and append each `newBridgeManager` to it.